skipper.go

```diff
@@ -1706,8 +1715,12 @@ func run(o Options, sig chan os.Signal, idleConnsCH chan struct{}) error {
 		o.CustomFilters = append(o.CustomFilters, compress)
 	}
 
-	if len(o.LuaModules) > 0 {
-		lua, err := script.NewLuaScriptWithOptions(script.LuaOptions{Modules: o.LuaModules})
+	if len(o.LuaModules) > 0 && len(o.LuaSources) > 0 {
```

Comment:
Here it should be OR condition. We have default config for lua in builtin filters and this one overrides it

Reply:
the condition I dropped, because that's not correct anymore, because default empty list and disable is []string{"none"}